web/crux/src/app/deploy/deploy.dto.ts

```diff
@@ -244,6 +244,12 @@ export class DeploymentLogPaginationQuery extends PaginationQuery {
   readonly try?: number
 }
 
+export class DeployStartDto {
```

Comment:
It should be `StartDeploymentDto` to line up with `CopyDeploymentDto`, `PatchInstanceDto`, etc.